nucypher/network/server.py

```diff
@@ -359,17 +297,18 @@ def receive_treasure_map():
         the treasure map by first validating the request and the received
         treasure map itself.
 
-        We set the datastore identifier as the HRAC iff the node is running
+        We set the datastore identifier as the HRAC if the node is running
         as a decentralized node. Otherwise, we use the map_id in
         federated mode.
         """
+
         if not this_node.federated_only:
-            from nucypher.policy.collections import SignedTreasureMap as _MapClass
+            from nucypher.policy.maps import SignedTreasureMap as _MapClass
         else:
-            from nucypher.policy.collections import TreasureMap as _MapClass
+            from nucypher.policy.maps import TreasureMap as _MapClass
 
         # Step 1: First, we verify the signature of the received treasure map.
-        # This step also deserializes the treasure map iff it's signed correctly.
```

Comment:
Was it a typo or "if and only if"?

Reply:
I think I was over-cautious, and it's just a typo. There's no need for mathematical precision in this context (in fact, "iff" would be wrong, since that would imply that if the map is correctly signed, it is *always* deserialized, which may not be the case).